### DISCUSSION TITLE
bulk delete [Jul 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2699] [Discord Threads]
> 🙋‍♂️ **super**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### You can use API. This still is at table level.   
#### https://apis.nocodb.com/#tag/DB-table-row/operation/db-table-row-bulk-delete-all  
#  
> 🙋‍♂️ **super**  
#### I use NocoDB as nocode app so it's too complicated for me/ but thx  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/994894082566541312)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
